src/modules/RootPart.rb

```diff
@@ -1259,8 +1255,6 @@ def MountPartitions(root_device_current)
         fstab_ref = arg_ref(fstab)
         crtab_ref = arg_ref(crtab)
         read_fstab_and_cryptotab(fstab_ref, crtab_ref, root_device_current)
-        fstab = fstab_ref.value
```

Comment:
@lslezak @jreidinger at this place I get a warning about a useless `crtab` assignment, yet we remove also `fstab` which is used in `if FstabUsesKernelDeviceNameForHarddisks(fstab)` belog